index.js

```diff
@@ -0,0 +1,165 @@
+import './style.css';
+// import { filter } from 'lodash';
+import { check } from './script.js';
+
+let todoLists = [
+  // {
+  //   description: 'Exercise regularly.',
+  //   completed: false,
+  //   index: 0,
+  // },
+  // {
+  //   description: 'Play with the Todo list.',
+  //   completed: false,
+  //   index: 1,
+  // },
+  // {
+  //   description: 'Check all the items in the todo list',
+  //   completed: false,
+  //   index: 2,
+  // },
+  // {
+  //   description: 'use pomodor clock regularly.',
+  //   completed: false,
+  //   index: 3,
+  // },
+];
```

Comment:
Kindly, remove all commented-out code in your file so that it looks more professional and clean.